app/src/main/java/co/touchlab/droidconandroid/shared/network/RefreshScheduleDataRequest.java

```diff
@@ -11,4 +12,7 @@
 public interface RefreshScheduleDataRequest {
     @GET("/dataTest/scheduleData/{conventionId}")
     Observable<Convention> getScheduleData(@Path("conventionId") Integer conventionId);
+
+    @GET("/dataTest/scheduleData/{conventionId}")
+    Call<Convention> getScheduleDataSync(@Path("conventionId") Integer conventionId);
```

Comment:
What do these Sync versions do? They dont appear to be used

Reply:
They were testing calls. Nothing else